tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -869,6 +875,51 @@ public function rememberTheAvailableSpaces(): void {
 		Assert::assertNotEmpty($spaces, "No spaces have been found");
 	}
 
+	public function getPropertiesOfAResource($properties, $fullUrl, $user): void {
```

Comment:
If there is already an implementation for this in core test helpers IMO, you can just use that from here too. Like the one in `WebDavHelper.phpL#141`.